getting-started-github/README.md

```diff
@@ -0,0 +1,37 @@
+
+Go ahead and add a file. It doesn't have to be code, Github supports uploading almost any type of file (.pdf,.py, .o, etc.). Once you have added a file to the repository folder, navigate back to Github Desktop. You need to **commit** your changes. Ideally, you want to commit ever so often when you are working on a project. Think of it like hitting "save" on a word document. That's not what really happens behind the scenes at all, but its a pretty basic analogy that lets you get past this initial hurdle. You will also have to enter a commit message on the desktop app. This isn't always required, but usually it helps your teammates know what changes you made. This might also come in handy if (when?) you accidentally save a bug, and need to **revert** back to an older version. Check out the "Where can I learn more?" section for a couple resouces for this.
+
+Once you have committed your changes, you should **push** them. **Pushing** is when you send the changes you have made to the remote git repository. Remember this order - first you **commit**, then you **push**.
+
+You're done! You just made a file, committed it, and pushed it to the remote repository. If you refresh the browser page that holds your online repo, you should see the file you made along with your unique commit message. 
+
+## How does my team work together?
+This is a little bit of an advanced topic, but its integral if you are working on a team. The obvious flaw with the method I described earlier is if multiple people attempt to push conflicting changes to the same repository at the same time. **Branches** solve this problem in a pretty elegant way. To understand them way better than I could ever explain, read [this](https://codesource.io/understanding-branching-in-git/). 
+
+For my teams, we have one repo and everyone is added on as a [collaborator](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository). Next, we have everyone make their own branch and we keep the **master** branch seperate. This makes sure no one pushes untested code directly to our current product. Everyone pushed to their own branch, and then makes a **pull request** to master. To learn how to do this, follow [this](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). This way, everyone can review the proposed changes before **merging** (combining) the changes with the main master branch. 
+
+Remember to **pull** changes from the remote branch to your local copy whenever your teammate makes a change. This ensures that everyone has the most up to date version of the code, and no one is working on an already solved bug. 
+
```

Comment:
I would include a section on merge conflicts so that if one comes up people won't freak out and be like what is all this >>>>>>> and <<<<<<<< and will know what to look up/have links to click on